src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1992,7 +1994,8 @@ namespace SurfaceGeometry {
             for (SurfNum = 1; SurfNum <= TotSurfaces; ++SurfNum) {
                 if (!Surface(SurfNum).HeatTransSurf) continue;                                               // ignore shading surfaces
                 if (Surface(SurfNum).ExtBoundCond > 0 && Surface(SurfNum).ExtBoundCond != SurfNum) continue; // interzone, not adiabatic surface
-                if (!Construct(Surface(SurfNum).Construction).TypeIsIRT) continue;
+                if (!Construct(Surface(SurfNum).Construction).TypeIsIRT)
+                    continue;
```

Comment:
I do not like this style where the IF statement is indented but not braced.  Can be visually deceiving.  I'm not going to hold up this PR for something like that, but when more changes are needed, please fix that.